assets/js/admin/admin.js

```diff
@@ -436,6 +436,7 @@ jQuery(function ($) {
 						} );
 
 						$( document ).on( 'click', '.ur-grids .ur-toggle-grid-content .ur-grid-selector', function() {
+							alert( 'yo' )
```

Comment:
@krishna-suwal Remove this alert. Please check your code before creating PR.

Reply:
@lekhnath005 I'm really sorry for this mistake. I will not repeat this kind mistake again.